frontend/src/scenes/feature-flags/featureFlagLogic.ts

```diff
@@ -456,7 +456,7 @@ export const featureFlagLogic = kea<featureFlagLogicType>([
             }
         },
         loadAllInsightsForFlag: () => {
-            values.featureFlag.rollback_conditions.forEach((condition, index) => {
+            values.featureFlag.rollback_conditions?.forEach((condition, index) => {
```

Comment:
Do we need to do anything below at L466?

Reply:
was thinking no because that action creates atleast an empty array